Tasks/Common/find-files-legacy/findfiles.legacy.ts

```diff
@@ -44,7 +44,7 @@ function extractPattern(patterns: string[], cwd: string): any {
     let excludePatterns: string[] = [];
 
     patterns.forEach(p => {
-        p.split(',').forEach(inputFilter => {
+        p.split(';').forEach(inputFilter => {
```

Comment:
> p.split(';').forEach(inputFilter => { [](start = 7, length = 38)

is findfiles legacy being used by our task only? if not, then removing comma as seperator might broke other guys calling this. Can we not somehow pass this from caller's end.